### ISSUE TITLE
Correcting inaccuracy data for COVID-19 cases in Hong Kong

### ISSUE THREAD

**Issue by sammyfung** [CSSEGISandData/COVID-19#547]
I go through recent commits of this GitHub responsibility and I found that the data collected for Hong Kong is **inaccuracy** which probably JHU CSSE choose an incorrect data source from China.

1. I recommend JHU CSSE to refer to the open data [1] provided by the Hong Kong Government (HKGov).
2. The data source, the Health Department (the data provider in the first hand) at HKGov are completely different than mainland China. "Hong Kong" or "Hong Kong SAR" should be used at "Country/Region" columns of all data files.

[1] https://data.gov.hk/en-data/dataset/hk-dh-chpsebcddr-novel-infectious-agent

**Comment by ShawTim** [comment]
Definitely. Hong Kong and China dont share anything in terms of data. @CSSEGISandData  is now referring the wrong source of data.
Plus, They have different borders, immigration departments, virus prevention strategies.

You can see US State has separated advisories on Hong Kong and China.
https://travel.state.gov/content/travel/en/traveladvisories/traveladvisories/hong-kong-travel-advisory.html
https://travel.state.gov/content/travel/en/traveladvisories/traveladvisories/china-travel-advisory.html

We need the data to help us to prevent COVID-19. Now the data of Hong Kong is combined into China cases. This confuses ppl and put them into the threat of infection.

**Comment by nguyenhychan** [comment]
By putting China as the "nation" for Hong Kong, this is making it difficult to group the data according to regions with separate jurisdictions. Please, revert to the original naming and don't try to cover up using whatever reasons like the WHO or so on.

**Comment by lwchau** [comment]
Hong Kong has its own health system and border control, it should not be treated as China

**Comment by DorisHWM** [comment]
Hong Kong is operating with her own system and has her own measure in handling COVID-19. To put Hong Kong in the same data pool as China, it will lead to a non-sampling error. Please consider and have a seperated record for Hong Kong. Thank you

**Comment by knuclechan** [comment]
Hong Kong and Macau have their own policy and measure to cope with the wuhan virus, which is considered a huge success when comparing to China and Europe. You shouldn't put them under China and made their effect being unseen.

**Comment by joelyu** [comment]
Hong Kong has her own immigration, quarantine and medical systems when it comes to managing the COVID-19 outbreak, grouping Hong Kong's cases under China is simply wrong and misleading, please revert back to the original state of presenting Hong Kong as a separate region.

**Comment by sasha-developer** [comment]
Hong Kong and Macau have their own border and difference measures to cope with the Wuhan Virus, and merging the data would not help making the data accurate or helpful but the reverse.

**Comment by victor-siu** [comment]
I agree. Please fix it asap for the sake of public health.

**Comment by sammyfung** [comment]
@CSSEGISandData I create a web scraper project which you can simply run it to retrieve the daily accumulated COVID-19 cases from Hong Kong Department of Health, and output a CSV line in your daily CSV report format. No excuse.

https://github.com/sammyfung/COVID-19-HK-DH

**Comment by sjmackenzie** [comment]
Good grief has @CSSEGISandData still not fixed this?

**Comment by Sania011** [comment]
Hong Kong has a completely different containment, isolation and public health system than Mainland China. It is imperative to separate out data for HongKong in order to show the effectiveness of different measures. @CSSEGISandData should fix it asap

**Comment by girak** [comment]
it's disappointing... since the Johns Hopkins COVID-19 map no longer outline the graph data separately / accurately for such an important international hub of Hong Kong... many, including myself, have closed the browser tab of this map.

Unfortunately, this means for the citizens of Hong Kong, this map will no longer serve as the go-to visual data tool.  

I myself, have switched to load another option, with detailed graphs for HK instead. like - 
https://www.worldometers.info/coronavirus/country/china-hong-kong-sar

**Comment by sjmackenzie** [comment]
@girak I believe these are the heads behind the ideologically motivated team building the Johns Hopkins COVID-19 website: https://systems.jhu.edu/about/team/. You might want to contact them and see if they're aware of the shitfuckery going down by @ryllau and his Johns Hopkins COVID-19 team.

Wikipedia uses https://www.worldometers.info/coronavirus/country/china-hong-kong-sar so it seems to be the proper canonical information source. Sadly many projects are starting to use this JH data hence, Hong Kong no longer shows up in these spin off projects.
